EddiCore/EDDI.cs

```diff
@@ -31,7 +30,9 @@ public class EDDI
         public bool SpeechResponderModalWait { get; set; } = false;
 
         private static bool started;
-        internal static bool running = true;
+ 
+        private static bool running = true;
+        public static bool IsRunning => running;
```

Comment:
Why not use a single auto-property here, e.g. `public static bool IsRunning {get; private set; } = true;`?

Reply:
Fixed.